examples/hello/src/operations.ts

```diff
@@ -11,7 +11,7 @@ export class Hello {
 
   @GetApi('/greeting/:user') // Serve this function from HTTP GET requests to the /greeting endpoint with 'user' as a path parameter
   @OperonTransaction()  // Run this function as a database transaction
-  static async helloTransaction(ctxt: TransactionContext<Knex>, user: string) {
```

Comment:
Is this just for OpenAPI to work? I'm a bit worried that adding another decorator `@ArgSource(ArgSources.URL)` here may add too much complexity to our initial example app. On the other hand, this is also a good example for people to write correct code.

Reply:
Yes, it's required for `OpenApiGenerator` to generate valid definitions. OpenAPI requires that all variables in the path are matched with named parameters.

Reply:
Yes, if we change the URL to behave better we can always relax the requirement.